### PR TITLE
More intuitive onboarding three words screen

### PR DESCRIPTION
**Pull request by justinvdm**
- Copy changes to example
- More robust word parsing

### CODE REVIEW

**Comment by justinvdm** [comment]
@Mitso @nathanbegbie ready for review

**Comment by justinvdm** [comment]
screen shot 2017-01-09 at 2 34 03 pm

File: src/views/Onboarding/ThreeWords/__tests__/ThreeWords-test.js [hunk 1]
```diff
@@ -20,5 +20,18 @@ describe('ThreeWords', () => {
...
+      shallow(createComponent({ profile: fakeProfile({ tags }) }))
+        .findWhere(uidEquals('formStep'))
+        .prop('paginationDisabled');
+
+    expect(paginationDisabled('')).toBe(true);
+    expect(paginationDisabled('foo bar')).toBe(true);
+    expect(paginationDisabled('foo bar baz quux')).toBe(true);
+    expect(paginationDisabled('foo bar baz')).toBe(false);
+    expect(paginationDisabled('foo,bar  baz')).toBe(false);
+  });
```
Comment (nathanbegbie) [inline src/views/Onboarding/ThreeWords/__tests__/ThreeWords-test.js]: @justinvdm Maybe include `foo, bar` as a test case - with a comma and a space combined?

**Comment by nathanbegbie** [comment]
@justinvdm Besides extra test, +1 from me

**Pull request merged**